web/app/js/components/ConfigureProfilesMsg.jsx

```diff
@@ -164,7 +222,7 @@ ConfigureProfilesMsg.propTypes = {
 };
 
 ConfigureProfilesMsg.defaultProps = {
-  showAsIcon: false
+  showAsIcon: false,
```

Comment:
it would be good to avoid changing unnecessary lines that don't have to do with the PR code.

Reply:
sorry didn't notice that my prettier changed it I guess